rocketc/tests/test_rocketChat.py

```diff
@@ -41,3 +50,160 @@ def test_request_rocket_chat(self):
 
         self.assertEqual(data_post, users)
         self.assertEqual(data_get, info)
+
+    # Mock create token method
+    @patch('rocketc.rocketc.RocketChatXBlock.create_token')
+    def test_login(self, mock_token):
+        """"""
```

Comment:
@andrey-canon what is your take on emtpy docstrings, better to leave them or remove them?